scripts/setup-k8s-master.sh.in

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -v
+set -e
```

Comment:
I like to put the following at the top of every bash file:
```
set -o errexit
set -o nounset
set -o pipefail
```

nounset can be a bit of a pain but also helps find bugs earlier.